docs/docs/command-line-flags.md

```diff
@@ -329,6 +329,9 @@ The Web Site flags control the behavior of Marathon's web site, including the us
     Expose the execution time per method via the metrics endpoint (/metrics) using code instrumentation.
     Enabling this might noticeably degrade performance but it helps finding performance problems.
     These measurements can be disabled with --disable_metrics. Other metrics are not affected.
+* <span class="label label-default">v1.6.0</span> `--metrics_averaging_window` (Optional. Default: 30 seconds):
```

Comment:
Nit: It should be 1.6.x.

Reply:
If I am right, the labels there indicate the version _since_ this is available, so I think that "since v.1.6.0" makes sense. Otherwise "v1.6" is closer to your suggestion, but I have a feeling that "v1.6.x" is a bit off from the rest of the document formatting.